protocols/protocols.go

```
@@ -66,7 +68,17 @@ func MapTCPProtocolHandlers(log interfaces.Logger, h interfaces.Honeypot) map[st
 		return tcp.HandleADB(ctx, conn, md, log, h)
 	}
 	protocolHandlers["tcp"] = func(ctx context.Context, conn net.Conn, md connection.Metadata) error {
+		conn.SetReadDeadline(time.Now().Add(500 * time.Millisecond))
```

don't ignore the error here